target_chains/ethereum/sdk/solidity/PythStructs.sol

```diff
@@ -30,4 +30,17 @@ contract PythStructs {
         // Latest available exponentially-weighted moving average price
         Price emaPrice;
     }
+
+    struct TwapPriceFeed {
+        // The price ID.
+        bytes32 id;
+        // Start time of the TWAP
+        uint startTime;
+        // End time of the TWAP
+        uint endTime;
+        // TWAP price
+        Price twap;
+        // Down slot ratio
+        uint32 downSlotsRatio;
```

Comment:
since unlike rust we don't have methods over structs i recommend documenting exactly what it is, and how it should be used.